tests/alias.spec.ts

```diff
@@ -0,0 +1,141 @@
+import { beforeAll, beforeEach, expect, test } from "@jest/globals";
+import { getDeploymentsList } from "../src/deployments";
+import {
+  listAliases,
+  setVercelToken,
+  assignAlias,
+  deleteAlias,
+  getAlias,
+  listDeploymentAliases,
+} from "../src";
+import { WrapperError } from "../src/utils/fetch";
+import { DeploymentList } from "../src/types/deployment";
+
+let dplList: DeploymentList | null = null,
+  dplError: WrapperError | null = null;
+
+beforeAll(async () => {
+  // set vercel token for tests
+  setVercelToken(process.env.VERCEL_TOKEN as string);
+
+  // get and set deployment ID
+  const { data, error } = await getDeploymentsList();
+  dplList = data;
+  dplError = error;
+});
+
+beforeEach(() => {});
```

Comment:
why do you need this beforeEach if you're doing beforeAll? As in why do you need to initialize this file with a function returning an empty object?